### PR TITLE
[DS]: 📆 Date picker

### PR DESCRIPTION
**Pull request by MorganeLecurieux**
[Spec](https://www.notion.so/getlago/Date-picker-4a64a548f3e74aeca681a589c2fa99af)
[Figma](https://www.figma.com/file/Pyhpgv1CAy6nGkl3X9aqy0/%E2%9B%B5%EF%B8%8FDocumentation-asset?node-id=2320%3A28683)
[Material](https://mui.com/x/api/date-pickers/date-picker/)

### CODE REVIEW

File: src/pages/__devOnly/DesignSystem.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <Typography variant="caption">Only visible in dev mode</Typography>
+      </PageHeader>
+      <Container>
+        <Form onSubmit={(e) => e.preventDefault()}>
+          <Typography variant="headline">Form</Typography>
+          <Block>
+            <DatePickerField name="date" label="Date Picker" formikProps={formikProps} />
+            <DatePickerField
+              name="date"
+              label="Date Picker with helper"
```
Comment (ansmonjol) [inline src/pages/__devOnly/DesignSystem.tsx:31]: Shall we use translations instead ?

**Review by ansmonjol — APPROVED** [review]
2 minor comment otherwise LGTM

File: src/pages/__devOnly/DesignSystem.tsx (hunk 1 above)
Comment (MorganeLecurieux) [reply src/pages/__devOnly/DesignSystem.tsx:31]: Nop, it's dev only, I won't bother Mike for this, it's not even a regular page

Comment (ansmonjol) [reply src/pages/__devOnly/DesignSystem.tsx:31]: Oh yes!!! My bad

**Pull request merged**